docs/submission/dictionary-overview.md

```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
+| Stage IA1 | Measured invasion of the stroma no greater than 3 mm in depth and no wider than 7 mm diameter.|
+| Stage IA2 | Measured invasion of stroma greater than 3 mm but no greater than 5 mm in depth and no wider than 7 mm in diameter.|
+| Stage IB | Clinical lesions confined to the cervix or preclinical lesions greater than Stage IA. All gross lesions even with superficial invasion are Stage IB cancers.|
+| Stage IB1 | Clinical lesions no greater than 4 cm in size.|
+| Stage IB2 | Clinical lesions greater than 4 cm in size.|
+| Stage IIA | No obvious parametrial involvement. Involvement of up to the upper two-thirds of the vagina.|
+| Stage IAB | Obvious parametrial involvement, but not into the pelvic sidewall.|
+| Stage IIIA | No extension into the pelvic sidewall but involvement of the lower third of the vagina.|
+| Stage IIIB | Extension into the pelvic sidewall or hydronephrosis or non-functioning kidney.|
+| Stage IVA | Spread of the tumour into adjacent pelvic organs.|
+| Stage IVB | Spread to distant organs.|
+Reference: [https://screening.iarc.fr/viaviliappendix1.php](https://screening.iarc.fr/viaviliappendix1.php)
+
+### Binet staging system
+
+| Stage | Description |
+|-------|-------------|
+| Stage A | Fewer than 3 groups of enlarged lymph nodes (lymphadenopathy) and a high white blood cell count |
+| Stage B | More than 3 groups of enlarged lymph nodes and a high white blood cell count 
+| Stage C | Enlarged lymph nodes or spleen, a high white blood cell count, and low red blood cell or platelet counts |
+Reference: Amin, Mahul B.. AJCC Cancer Staging Manual, Eighth Edition (p. 975). American College of Surgeons.
+
+
+## Tumour grading classifications
+
+
+### Two-tier grading system
+
+| Grade |
+|-------|
+| Low grade |
+| High grade | 
+Reference: Reference: Amin, Mahul B.. AJCC Cancer Staging Manual, Eighth Edition
+
+
+### Three-grade system
+| Grade | Definition |
+|-------|------------|
+| GX | Cannot be assessed |
+| G1 | Well differentiated |
+| G2 | Moderately differentiated |
+| G3 | Poorly differentiated, undifferentiated |
+Reference: Reference: Amin, Mahul B.. AJCC Cancer Staging Manual, Eighth Edition
+
+
+### Four-tier grading system
+| Grade | Definition |
+|-------|------------|
+| GX | Grade cannot be assessed |
+| G1 | Well differentiated |
+| G2 | Moderately differentiated |
+| G3 | Poorly differentiated |
+| G4 | Undifferentiated |
+Reference: Reference: Amin, Mahul B.. AJCC Cancer Staging Manual, Eighth Edition
+
+
+### Grading for Gastrointestinal Stromal Tumours (GISTs)
+	
+| Mitotic Rate | Definition |
+|--------------|------------|
+| Low | 5 or fewer mitoses per 5 mm2|
+| High | Over 5 mitoses per 5 mm2 |
+Reference: Amin, Mahul B.. AJCC Cancer Staging Manual, Eighth Edition (p. 546). American College of Surgeons.
+
+
+### Grading system for GNETs
```

Comment:
for rest of headings, just to be consistent with how this page was up top,  capitalize the important words